migrations/1584819325407-runIndexSync.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['mongoose' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5221564)